### ISSUE TITLE
optimizeSvgEncode unclosed string

### ISSUE THREAD

**Issue by gdub22** [postcss/postcss-url#105]
Thanks for the optimizeSvgEncode feature.

It appears to be creating invalid css:
`Module build failed: Unclosed string`
```css
background-image: url(data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' width='32' height='32' viewBox='0 0 32 32'%3e%3cpath d='M0 4v24h32V4H0zm6 22H2v-4h4v4zm0-8H2v-4h4v4zm0-8H2V6h4v4zm18 16H8V6h16v20zm6 0h-4v-4h4v4zm0-8h-4v-4h4v4zm0-8h-4V6h4v4zm-18 0v12l8-6z'/%3e%3c/svg%3e);
```

Look like it's because Inlining inserts without wrapping the whole thing in quotes and there are now unencoded quotes in the svg when using optimizeSvgEncode

**Comment by gdub22** [comment]
Looks like this retains the input css's quoting, so this works ok:

```css
.style {
  background-image: url("./image.svg");
}
```

But this fails:

```css
.style {
  background-image: url(./image.svg);
}
```

Quotes on file path urls are optional according to the css spec, but maybe this module should enforce adding quotes when inlining.

**Comment by sergcen** [comment]
@gdub22 thanks for feedback. Yes, you're right. I will try fix it soon.

**Comment by Realetive** [comment]
replace https://github.com/postcss/postcss-url/blob/master/src/type/inline.js#L81 on 
```
return `'${(options.includeUriFragment && asset.hash) ? encodedStr + asset.hash : encodedStr}'`;
```
work for me.

**Comment by sergcen** [comment]
fixed here [#7.1.1](https://github.com/postcss/postcss-url/releases/tag/7.1.1)

**Comment by gdub22** [comment]
Thank you. Confirmed it's working.
